src/types/index.ts

```
@@ -0,0 +1,52 @@
+export type ViewMode = "tcg" | "simu";
+
+export interface DiceInstance {
+  key: number;
+  roll: number;
+  animationClass: "odd-roll" | "even-roll";
+}
+
+// イベント型
+export interface AppEvents {
+  "open-dice-modal": void;
+  "reset-state-notification": void;
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)